### PR TITLE
Add new virtual integration for opower City of Austin Utilities provider

### PR DESCRIPTION
**Pull request by max2697**
## Proposed change

Add new virtual integration for opower City of Austin Utilities provider
Related to PR https://github.com/home-assistant/home-assistant.io/pull/30840 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/108337
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5114
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
This PR is missing a PR upstream in our brands repository.

Please create a PR there and link it up with the parent PRs here and in Core.

**Comment by max2697** [comment]
@frenck Created new PR for brands: https://github.com/home-assistant/brands/pull/5114

File: source/_integrations/coautilities.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: City of Austin Utilities
+description: Get energy usage from City of Austin Utilities using the Opower integration
+ha_category:
+  - Energy
+  - Sensor
+ha_release: 2024.2
+ha_domain: oru_opower
```
Comment (tronikos) [inline source/_integrations/coautilities.markdown]: update

Comment (max2697) [reply source/_integrations/coautilities.markdown]: Fixed

**Review by tronikos — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @max2697 👍 

../Frenck

**Pull request merged**